<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaSecundaria</class>
 <widget class="QDialog" name="JanelaSecundaria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="numero_pagina">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Janela Secundária</string>
   </property>
  </widget>
  <widget class="QLabel" name="Imagem_github">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Imagens/github-logo.webp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Mudar_Imagem">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #4CAF50; /* Cor de fundo do botão */
  color: white; /* Cor do texto */
  padding: 10px 20px; /* Espaçamento interno do botão */
  border: none; /* Remover borda padrão */
  border-radius: 15px; /* Arredondar bordas */
  font-size: 16px; /* Tamanho da fonte */</string>
   </property>
   <property name="text">
    <string>Mudar Imagem</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>125</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Mostrar Seleções</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
